### DISCUSSION TITLE
Is it possible to disable the graph in the Discord embed?

### DISCUSSION THREAD

**Discussion by FelixKawaii** [a-sync/game-server-watcher#11] [Q&A] [answered]
The graph is cool but I'd prefer a cleaner look :)

**Comment by a-sync** [comment, marked as answer]
Yes. It is now (⌐■_■)  
Just update to the latest version and look for the `Show graph` checkbox.  
![image](https://github.com/a-sync/game-server-watcher/assets/14183614/5688a1d2-f58b-4ca5-940f-971650bebef5)

**Reply by FelixKawaii** [reply]
Works perfectly, thank you very much <3
